services/galley/src/Galley/Data.hs

```diff
@@ -204,7 +205,10 @@ teamConversations t =
     <$> retry x1 (query Cql.selectTeamConvs (params Quorum (Identity t)))
 
 teamMembers :: forall m. (MonadThrow m, MonadClient m) => TeamId -> Range 1 HardTruncationLimit Int32 -> m ([TeamMember], Bool)
-teamMembers t (fromRange -> limit) = do
+teamMembers t = teamMembers' t . Just
+
+teamMembers' :: forall m. (MonadThrow m, MonadClient m) => TeamId -> Maybe (Range 1 HardTruncationLimit Int32) -> m ([TeamMember], Bool)
+teamMembers' t (maybe hardTruncationLimit fromRange -> limit) = do
```

Comment:
All uses of this function send it `Nothing`, can we just call it `teamMembersWithDefaultLimit` and not take the `Maybe`?

I am a bit uncertain about adding a `'` to a function, it usually doesn't express intent very well. Unless I am missing some context.